README.md

```diff
@@ -160,38 +178,35 @@ protected void onDestroy() {
     super.onDestroy();
 }
 ```
+## (Optional) Adding attachments to your beacons
 
```

Comment:
Add explanation what attachments are:

While zone identification is based on tags, **attachments** are a way to add additional content to a beacon and a zone it defines. Think of it as a custom backend where you can assign any additional data to a particular beacon.